department-of-veterans-affairs/vets-api
# Fixes 'veteranReadinessAndEmployment' in form_data.rb to match 200995…

## Description of change
ensure all SC endpoints are functioning as expected in the staging environment

## Original issue(s)
[API-11992](https://vajira.max.gov/browse/API-11992)

## Things to know about this PR
Manual Tests:
Success Use Cases
Use Case 1:

- retrieve Contestable Issues for a SC disability appeal
- use 1 contestable issue from the returned list to create and submit a SC
- upload 1 document of evidence associated to the submitted SC

Use Case 2:

- retrieve Contestable Issues for a SC disability appeal
- create 1 write-in contestable issue 
- provide information for medical records to request
- create and submit the SC

Use Case 3:

- retrieve Contestable Issues for a SC disability appeal
- use 10 contestable issue from the returned list and/or write-in Issues to create and submit a SC
- upload 1 document of evidence associated to the submitted SC
- ensure additional pages are generated on the PDF
- Error Use Cases

Use Case 4:

- retrieve Contestable Issues for a SC disability appeal
- leave at least one required field blank
- receive API validation error

Use Case 5:

- retrieve Contestable Issues for a SC disability appeal
- Choose Compensation LOB and do not select yes for the 5103 Notice field
- receive custom API validation error with link to 5103 Guidance

Use Case 4:

- retrieve Contestable Issues for a SC disability appeal
- use 1 contestable issue from the returned list to create and submit a SC
- Do not upload any documents - I know this won't actively error, but maybe we just log that no document was uploaded?
